### PR TITLE
Fixed label width changes

### PR DESCRIPTION
**Pull request by kjyv**
The label now has a fixed size that never changes by padding the text with as much is necessary to get a certain pixel width. It is not always pixel perfect but mostly looks alright. I also took over the changing icons from NaWer's fork.

### CODE REVIEW

File: indicator-netspeed.c [hunk 1]
```diff
@@ -30,21 +32,40 @@ GtkWidget *net_down_item;
...
     }
     else
     {
-        string = g_strdup_printf("%.2f MiB/s", data/1000000.0);
+        string = g_strdup_printf("%.2f MiB/s", data/1048576.0);
     }
+
+    if(padding)
+    {
+        //render string and get it's pixel width
```
Comment (mgedmin) [inline indicator-netspeed.c:61]: Typo: its

File: README.md [hunk 2]
```diff
@@ -8,21 +8,18 @@ Usage
 
 ```
 sudo apt-get install build-essential libgtop2-dev libgtk-3-dev libappindicator3-dev git-core
-git clone git://github.com/mgedmin/indicator-netspeed.git
+git clone git://github.com/kjyv/indicator-netspeed.git
```
Comment (mgedmin) [inline README.md:11]: This seems strange in a pull request.

**Comment by mgedmin** [comment]
Congratulations, you're now a repo collaborator!  Feel free to merge this.  Also, please add yourself to the credits section in the README.

I can't test the changes since I no longer use Unity, so I won't be merging this myself.

**Comment by kjyv** [comment]
Hah, alright, easier that way :)
I guess I was being sloppy with my pull request and the url change (the old url didn't make sense on my fork page). In the end, maybe the github url should not be in the README.md anyway.
I've since added some more things to my fork so I will just merge the rest in as well.

**Comment by netikras** [comment]
one does not have to be using unity to use this app.... I'm on xfce4 and it works fine

**Pull request closed without merging**